### PR TITLE
text changes

### PR DESCRIPTION
**Pull request by AndySheu**
I edited the text, hopefully for the better. Slightly more content has
been added.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -7,25 +7,32 @@ image: about-background
 
 ## Who are we?
 
-Iron Claw is a [First Robotics Competition](http://www.usfirst.org/roboticsprograms/frc) (FRC)
-team run by the Los Gatos High School Robotics Club. Our team number is 972 and our rookie year is 2016!
-We hope to see all of our fellow teams at Silicon Valley Regional in April!
+Iron Claw is a <a href="http://www.usfirst.org/roboticsprograms/frc">FRC</a>
```
Comment (Joshua-Anderson) [inline about.md]: This is a markdown page. The proper format for links is `[text](http://example.com)`.

File: about.md [hunk 2]
```diff
@@ -7,25 +7,32 @@ image: about-background
...
-Unfortunately, team 972 stopped competing in 2008 due to a lack of interest and the retirement of the head mentor.
+Unfortunately, team 972 stopped competing in 2008 due to a lack of interest and the retirement of the lead mentor.
 Students traveled to Pioneer High School's team 668, the [Apes of Wrath](http://www.apesofwrath668.org")
 to continue participating in robotics.
 
-Soon, enough interest built up in LGHS for the Robotics Club to form. We participated as team 6730 in the
-FIRST Tech Challenge (FTC) in 2012. FTC is a smaller-scale and lower-cost robotics competition. In 2014, the
-program grew enough for LGHS to field a second FTC team, 9097. We still participated in FRC with Pioneer
-High School.
+Soon, enough interest built up for the Robotics Club to form at LGHS. We participated as team 6730 in the
```
Comment (Joshua-Anderson) [inline about.md]: Maybe `reform` rather then `form`

File: about.md [hunk 3]
```diff
@@ -7,25 +7,32 @@ image: about-background
...
+FIRST Tech Challenge (FTC) in 2012. FTC is a smaller-scale and lower-cost robotics competition. In fall of
+2014, the program grew enough for LGHS to field a second FTC team, 9097. At the end of the 2015 FRC season,
+the Los Gatos High School Robotics Club decided to split off from Pioneer High School and the Apes of Wrath
+due to many students' inability to transport on a daily basis and our team's growing size.
```
Comment (Joshua-Anderson) [inline about.md]: `commute` may be a better word than `transport`

File: about.md [hunk 4]
```diff
@@ -7,25 +7,32 @@ image: about-background
...
 
-We are now restarting team 972 as "Iron Claw". Our mission is to grow interest in STEM in the Los Gatos community
+We are now restarting team 972 as "Iron Claw", featuring a fresh, new team while still retaining characteristics
```
Comment (Joshua-Anderson) [inline about.md]: `featuring a fresh, new team while still retaining characteristics of our old team` sounds odd to me. Maybe remove or rephrase this

**Comment by Joshua-Anderson** [comment]
Some of this overlaps with the [main page](https://github.com/iron-claw-972/iron-claw-972.github.io/blob/master/index.html#L17-L19), so it needs to updated as well.

Besides my comments it looks good to me. :+1: Thanks!

**Comment by Joshua-Anderson** [comment]
Assigning @gautam-prab to grammar check all the things.

**Comment by gautam-prab** [comment]
Ok, I committed for some basic grammar improvements.

**Comment by gautam-prab** [comment]
Sorry, forgot to update the main index

File: about.md [hunk 5]
```diff
@@ -7,25 +7,28 @@ image: about-background
...
 
-We are now restarting team 972 as "Iron Claw". Our mission is to grow interest in STEM in the Los Gatos community
-and to teach our members real-life STEM skills, teamwork, and
+We are now restarting team 972 as "Iron Claw". Our mission is to develop and nuture interest in 
+STEM within the Los Gatos community and to teach our members real-life STEM skills, teamwork, and
 [gracious professionalism](http://www.usfirst.org/aboutus/gracious-professionalism).
 
-You can find our team's old website [here](http://losgatosmanufacturing.com/clubs/robotics_team.php").
+Our team's former website can still be found [here](http://losgatosmanufacturing.com/clubs/robotics_team.php").
+Our progress through the years can also be found at the[Blue Alliance](https://www.thebluealliance.com/team/972"), a website that keeps team information, event results, and video from previous years for all FRC teams.
```
Comment (Joshua-Anderson) [inline about.md]: Space needed at `the[Blue Alliance]`

**Comment by Joshua-Anderson** [comment]
`fellow teams and students` feels a little redundant.

File: index.html [hunk 6]
```diff
@@ -14,9 +14,9 @@
       <h2 class="mdl-card__title-text">Who are we?</h2>
     </div>
     <div class="mdl-card__supporting-text">
-      Iron Claw is a <a href="http://www.usfirst.org/roboticsprograms/frc">First Robotics Competition</a> (FRC)
-      team run by the Los Gatos High School Robotics Club. Our team number is 972 and our rookie year is 2016!
-      We hope to see all of our fellow teams at Silicon Valley Regional in April!
+      Iron Claw is an FRC team run by the Los Gatos High School Robotics Club.
+      This year (2016), we will restart team 972 at LGHS.
```
Comment (Joshua-Anderson) [inline index.html]: I'm not sure we need `2016` here. I would either add 2016 as part of the sentence or remove it entirely.

**Comment by Joshua-Anderson** [comment]
We don't explain what FRC is anywhere. Maybe we want to introduce it somewhere?

**Comment by Joshua-Anderson** [comment]
I fixed up the things I saw. @iron-claw-972/owners Does anyone have time to review this?

**Pull request merged**